wcbench.sh

```diff
@@ -214,6 +216,69 @@ install_cbench()
     fi
 }
 
+install_cbench_fedora()
```

Comment:
We're adding a ton of unnecessary code duplication buy splinting this into Ubuntu and Fedora functions, since (I think) the only part that's OS-specific is the `yum` vs `apt-get` install step. So, I suggest not creating a different function and instead adding a conditional to check which OS we're on and use the correct install command.